pipelines/ci-integration.Jenkinsfile

```diff
@@ -0,0 +1,232 @@
+
+                    legionVersion = map["LEGION_VERSION"]
+
+                    print "Loaded version ${legionVersion}"
+
+                    if (!legionVersion) {
+                        error 'Cannot get legion release version number'
+                    }
+                }
+            }
+        }
+
+        stage('Terminate Cluster if exists') {
+            steps {
+                script {
+                    result = build job: env.param_terminate_cluster_job_name, propagate: true, wait: true, parameters: [
+                        [$class: 'GitParameterValue', name: 'GitBranch', value: env.mergeBranch],
+                        string(name: 'LegionInfraVersion', value: legionInfraVersion),
+                        string(name: 'ClusterName', value: env.param_cluster_name),
+                        string(name: 'LegionProfilesBranch', value: env.param_legion_profiles_branch),
+                        string(name: 'CicdRepoGitBranch', value: env.param_legion_cicd_branch)
+                    ]
+                }
+            }
+        }
+
+        stage('Create Cluster') {
+            steps {
+                script {
+                    result = build job: env.param_create_cluster_job_name, propagate: true, wait: true, parameters: [
+                        [$class: 'GitParameterValue', name: 'GitBranch', value: env.mergeBranch],
+                        string(name: 'ClusterName', value: env.param_cluster_name),
+                        string(name: 'LegionInfraVersion', value: legionInfraVersion),
+                        string(name: 'LegionProfilesBranch', value: env.param_legion_profiles_branch),
+                        string(name: 'CicdRepoGitBranch', value: env.param_legion_cicd_branch)
+                    ]
+                }
+            }
+        }
+
+        stage('Deploy Legion & run tests') {
+            steps {
+                script {
+                    result = build job: env.param_deploy_legion_job_name, propagate: true, wait: true, parameters: [
+                        [$class: 'GitParameterValue', name: 'GitBranch', value: env.mergeBranch],
+                        string(name: 'ClusterName', value: env.param_cluster_name),
+                        string(name: 'LegionVersion', value: legionVersion),
+                        string(name: 'LegionInfraVersion', value: legionInfraVersion),
+                        string(name: 'TestsTags', value: env.param_tests_tags ?: ""),
+                        string(name: 'commitID', value: env.commitID),
+                        booleanParam(name: 'DeployLegion', value: true),
+                        booleanParam(name: 'UseRegressionTests', value: true),
+                        string(name: 'LegionProfilesBranch', value: env.param_legion_profiles_branch)
+                    ]
+                }
+            }
+        }
+    }
+
+    post {
+        always {
+            script {
+                if (env.param_remainTheCluster.toBoolean()) {
```

Comment:
Are you really sure that `param_remainTheCluster` parameter works as expected? And rename it to `param_remain_the_cluster` :D

Reply:
there is a condition based on the parameter. Else section deletes the cluster.
parameter was renamed as requested.

Reply:
done